actions.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from PyQt5 import QtWidgets
+
+import streamlink
+from constants import MUTE_CHECKBOX
+from containers import LiveStreamContainer
+
+
+def mute_all_streams(parent):
```

Comment:
Fuck me, now i get it 😆 

Constantly got this error when using methods on `ApplicationWindow`:
```
self.actions[MUTE_CHECKBOX].toggled.connect(self.mute_all_streams)
TypeError: argument 1 has unexpected type 'QAction'
```
Strange, I don't pass in a QAction? Hmm.. Never quite figured out what it was, figured there was some kind of dark Qt magic going on in the background, and made the above solution in despair.

The root cause? 
The name of the `QAction` in the `.ui` file is `mute_all_streams`. *facepalm*
If I had just stopped and thinked for a second the above error actually made sense... 🤣